src/app/website/company-home/feature-summary/feature-summary.component.scss

```diff
@@ -0,0 +1,154 @@
+@import '../../shared/bits';
+
+
+.featuresummary{
+  overflow-x:hidden;
+}
+
+.wrap{
+    display: flex;
+    flex-wrap:wrap;
+}
+.featuresummary__content-img{
+
+  margin-left:2%;
+  margin-right:2%;
+  height: 250px;		
+  width:500px;
+  box-shadow: 0 2px 4px 0 rgba(0,0,0,0.5);
+  @include breakpoint (phablet){
+    margin-left:10%;
+    margin-right:10%;
+  }
+}
+.featuresummary__content-heading{
+  margin-left: 1%; 
+  text-align: left;
+  color:$black;
+  font-family:$fontBitter;	
+  font-size:$big;
+  line-height: 28px;
+  @include breakpoint (phablet){
+    text-align: center;
+  }
+  
+}
+.featuresummary__content-description{
+  margin-left: 1%;
+  text-align: left;
+  color:$black;	
+  font-family:$fontRaleway;
+  font-size: $medium;	
+  line-height: 19px;
+  @include breakpoint (phablet){
+    text-align: center;
+  }
+  
+}
+.featuresummary__content-list{
+  line-height: 28px;
+  @include breakpoint (phablet){
+    text-align: center;
+  }
+}
+.featuresummary__content-advantage{
+
+  padding-bottom: 10px;
+  padding-top:10px;
+  display: inline-block;
+  color:$black;
+  font-family:$fontRaleway;
+  font-size: $medium;
+  line-height: 19px;
+  font-family: $fontRaleway;
+ 
+}
+
+.featuresummary__content:nth-child(odd){
```

Comment:
why nth-child? It is not required. you use simple `odd` and `even` clases as you are in for-loop